### ISSUE TITLE
COVID LOINC Group 36662633

### ISSUE THREAD

**Issue by xj2193** [OHDSI/Vocabulary-v5.0#627]
We noticed the new LOINC group 36662633 have 102 descendants Lab Tests. However only 80 are included in the descendants of 756055. Please see the following queries. Could you let me know if there's anything that I missed or it's an issue needs to be fixed? Thanks very much!

select distinct *
from concept_ancestor ca
where ca.ancestor_concept_id=756055
and descendant_concept_id in 
(select descendant_concept_id from concept_ancestor 
where ancestor_concept_id = 36662633)

**Comment by cukarthik** [comment]
Hi @TinyRickC137 ,

I believe this is an issue b/c LOINC has a group ([36662633](https://athena.ohdsi.org/search-terms/terms/36662633)) to classify SAR-COV2. OHDSI created a grouping when one did not exist using an OMOP Extension code ([756055](https://athena.ohdsi.org/search-terms/terms/756055)). If I run this query, I see 24 standard LOINC codes in the Measurement domain that are not part of parent code ([756055](https://athena.ohdsi.org/search-terms/terms/756055)):

`
select distinct *
from concept_ancestor ca
where ca.ancestor_concept_id=36662633
and descendant_concept_id not in
(select descendant_concept_id 
from concept_ancestor as ca2
join concept as c on c.concept_id=ca2.descendant_concept_id
where ca2.ancestor_concept_id = 756055 and c.standard_concept='S' and c.domain_id='Measurement')
`

@mik-ohdsi  can you confirm?

**Comment by MariaRohozhkina** [comment]
There is no division between SARS-COV2 and SARS-related coronavirus in LOINC hierarchy. But we separate it, because SARS-related coronavirus is group of viruses not only SARS-COV2, so part of LOINC group 36662633 has another hierarchy. Please, take descendants of 705104 and they should appear.  
Also, we don't include panels, since they are not used for coding and are not consistent in composition.

**Comment by mik-ohdsi** [comment]
So, @cukarthik and @MariaRohozhkina. Is the conclusion that we should clean up a little and retire that OMOP Extension concept now and map it over to a better one from LOINC, updating / adding mappings to that grouping concept we chose from that fairly decent LOINC concept list for [SARS CoV2 detection](https://loinc.org/LG51017-8/) plus more? Or is our finding that there still is no good replacement for that OMOP Extension? And can you help me here, as I unfortunately did not fully understand how the [general Coronavirus detection OMOP extension concept](https://athena.ohdsi.org/search-terms/terms/705104) is helping us. I probably did not understand how they differ when compared to the LOINC hierarchy. Thanks!

**Comment by Alexdavv** [comment]
> retire that OMOP Extension concept now and map it over to a better one from LOINC

No. The LOINC grouper is only for LOINC concepts. By the definition of the ontology we build, LOINC can't be above the SNOMED and others.

> from that fairly decent LOINC concept list

It's not. The ones we excluded are not exactly the SARS-2 detection.

> Or is our finding that there still is no good replacement for that OMOP Extension?

Right. Only SNOMED could have it. Or OMOP Extension.

> [general Coronavirus detection OMOP extension concept](https://athena.ohdsi.org/search-terms/terms/705104) is helping us

If you want not just SARS-2, but also SARS-1 and others, you pick it.

> how they differ when compared to the LOINC hierarchy

Look at this concept [SARS-related coronavirus E gene [Presence] in Saliva (oral fluid) by NAA with probe detection](https://athena.ohdsi.org/search-terms/terms/36031814). It's not exactly about the SARS-2.
![image](https://user-images.githubusercontent.com/32357692/168078383-0683dc3a-c8df-4e0a-8e46-56b7d909ca4d.png)

And it's not even linked to the SARS-2 directly.
![image](https://user-images.githubusercontent.com/32357692/168078515-8a46aa2b-4852-4348-9c21-f4933577454c.png)


But because the [panels](https://athena.ohdsi.org/search-terms/terms/1617448) are not consistent... 
![image](https://user-images.githubusercontent.com/32357692/168078644-9bd48c78-5fe8-4af3-9c7f-1702c842ae21.png)

...it rolls up to the SARS-2.
![image](https://user-images.githubusercontent.com/32357692/168078800-4bd67469-b548-4852-b61a-4fe58a11f522.png)
